apps/client/package.json

```diff
@@ -25,6 +25,7 @@
   "devDependencies": {
     "@typescript-eslint/eslint-plugin": "^6.7.2",
     "@typescript-eslint/parser": "^6.7.2",
+    "cypress": "^13.3.0",
```

Comment:
How should I run the tests? Couldnt found any test related script in package.json